tests/platform_tests/cli/test_show_chassis_module.py

```diff
@@ -85,21 +94,57 @@ def test_show_chassis_module_status(duthosts, enum_rand_one_per_hwsku_hostname,
             if mod_idx in skip_logical_lc_list:
                 continue
             else:
-                pytest_assert(res[mod_idx]['Oper-Status'] == 'Empty',
-                              "Oper-status for slot {} should be Empty but it is {}".format(
-                                  mod_idx, res[mod_idx]['Oper-Status']))
+                pytest_assert(
+                    res[mod_idx]['Oper-Status'] == 'Empty',
+                    (
+                        "Oper-status for slot {} should be Empty but it is {}. "
+                        "Expected module to be empty, but it may be in a different state. "
+                        "Module details: {}"
+                    ).format(
+                        mod_idx,
+                        res[mod_idx]['Oper-Status'],
+                        res[mod_idx]
+                    )
+                )
+
         else:
-            pytest_assert(res[mod_idx]['Oper-Status'] == 'Online',
-                          "Oper-status for slot {} should be Online but it is {}".format(
-                              mod_idx, res[mod_idx]['Oper-Status']))
+            pytest_assert(
+                res[mod_idx]['Oper-Status'] == 'Online',
+                (
+                    "Oper-status for slot {} should be Online but it is {}. "
+                    "Module details: {}"
+                ).format(
+                    mod_idx,
+                    res[mod_idx]['Oper-Status'],
+                    res[mod_idx]
```

Comment:
The original summary is pretty good, not sure how large is res[mod_idx], it's better to keep the original one.